src/runtime/pkg/oci/utils.go

```diff
@@ -475,6 +475,16 @@ func addHypervisorConfigOverrides(ocispec specs.Spec, config *vc.SandboxConfig,
 		}
 	}
 
+	if value, ok := ocispec.Annotations[vcAnnotations.GuestHookTimeout]; ok {
```

Comment:
This should use the `newAnnotationConfiguration().setInt()`, please refer to the code around here.